datadir/default.properties

```diff
@@ -117,3 +117,7 @@ smtpHost=smtp
 # SMTP server domain name
 # Listening port of the SMTP server
 smtpPort=25
+
+
+# Header Authentication
+headerAuthentication=true
```

Comment:
Not necessary as long as you are activating the right profile for the gateway (see https://github.com/georchestra/georchestra-gateway/blob/main/datadir/gateway/security.yaml#L104-L114).

I also fear it could break the other docker compositions making use of this datadir.